### PR TITLE
Color standards

### PR DESCRIPTION
**Pull request by gloddy**
- Added new semantic colors
- Replaced brown, orange, green with semantic draft, scheduled, published colors
- delete unused brown, orange, green colors
- semantic toolbar view colors applied to toolbar

screen shot 2016-08-19 at 1 29 48 pm

### CODE REVIEW

File: styleguide/pane.scss [hunk 1]
```diff
@@ -135,11 +135,11 @@ $toolbar-height: 48px;
     }
 
     &.valid svg {
-      fill: $green;
+      fill: $published;
     }
 
     &.warnings svg {
-      fill: $orange;
+      fill: $published;
```
Comment (nelsonpecora) [inline styleguide/pane.scss]: ↑

File: styleguide/pane.scss [hunk 2]
```diff
@@ -335,7 +335,7 @@ $schedule-item-width: calc(50% - 8px);
 }
 
 .kiln-toolbar-pane .schedule-publish {
-  @include button-outlined($orange, $white);
+  @include button-outlined($published, $white);
```
Comment (nelsonpecora) [inline styleguide/pane.scss]: ↑

File: styleguide/pane.scss [hunk 3]
```diff
@@ -396,13 +396,13 @@ $schedule-item-width: calc(50% - 8px);
 
 // special color for warnings
 .kiln-toolbar-pane .publish-warning .label {
-  color: $orange;
+  color: $published;
```
Comment (nelsonpecora) [inline styleguide/pane.scss]: ↑

File: styleguide/pane.scss [hunk 4]
```diff
@@ -396,13 +396,13 @@ $schedule-item-width: calc(50% - 8px);
...
 }
 
 .kiln-toolbar-pane .warning-message {
   @include primary-text();
 
-  color: $orange;
+  color: $published;
```
Comment (nelsonpecora) [inline styleguide/pane.scss]: ↑

File: styleguide/toolbar.scss [hunk 5]
```diff
@@ -14,7 +14,7 @@ $height: 48px;
 }
 
 .kiln-toolbar {
-  background-color: $black-60;
+  background-color: $toolbar-view;
```
Comment (nelsonpecora) [inline styleguide/toolbar.scss]: shouldn't this be `$toolbar-edit`?

File: styleguide/toolbar.scss [hunk 6]
```diff
@@ -40,15 +40,15 @@ body {
 }
 
 .kiln-toolbar-inner .kiln-toolbar-button, {
-  @include button-filled-large($black-60);
+  @include button-filled-large($toolbar-view);
 
   flex: 0 0 auto;
-  border-left: 1px solid $black-50;
+  border-left: 1px solid $toolbar-view-border;
```
Comment (nelsonpecora) [inline styleguide/toolbar.scss]: same here

File: styleguide/toolbar.scss [hunk 7]
```diff
@@ -148,7 +148,7 @@ body {
 }
 
 .kiln-toolbar-inner .scheduled {
-  @include button-filled-large($orange);
+  @include button-filled-large($published);
```
Comment (nelsonpecora) [inline styleguide/toolbar.scss]: ↑

File: styleguide/toolbar.scss (hunk 5 above)
Comment (gloddy) [inline styleguide/toolbar.scss]: Depends if you want the lighter toolbar now or want to wait until the Edit/View toggle is in place.
I chose to leave the darker grey for now.

Comment (nelsonpecora) [inline styleguide/toolbar.scss]: ok

**Comment by nelsonpecora** [comment]
👍

**Comment by gloddy** [comment]
@nelsonpecora Fixed the Orange ->Scheduled mistake. Defaulting to darker View colors for the toolbar until the View/Edit toggle comes onto the scene.

**Pull request merged**